packages/request-node/package.json

```diff
@@ -47,7 +47,7 @@
     "@requestnetwork/smart-contracts": "0.28.0",
     "@requestnetwork/types": "0.35.0",
     "@requestnetwork/utils": "0.35.0",
-    "@truffle/hdwallet-provider": "1.2.3",
+    "@truffle/hdwallet-provider": "2.1.1",
```

Comment:
Please revert the hdwallet-provider version change if it is not needed.

Reply:
will do